scipy/stats/_qmc.py

```diff
@@ -968,11 +971,11 @@ class Sobol(QMCEngine):
     MAXBIT = _MAXBIT
 
     def __init__(self, d, scramble=True, seed=None):
+        super().__init__(d=d, seed=seed)
```

Comment:
Is this only needed in `Sobol`?

Reply:
The others were doing this already at the beginning of the init so it's fine. I would need to do some adjustments for the multivariate ones now that I re-read the code a bit, but it's unrelated to this.